uc-crux-llvm/src/UCCrux/LLVM/Classify/Types.hs

```diff
@@ -214,13 +227,36 @@ ppUnfixable =
 data Unfixed
   = UnfixedArgPtrOffsetArg
   | UnfixedFunctionPtrInArg
+  | -- The following would be addressed by applying "missing precondition"
+    -- heuristics to return values from skipped functions
+    UnfixedRetReadUninitialized
+  | UnfixedRetWriteUnmapped
+  | UnfixedRetCall
+  | UnfixedRetMemset
+  | UnfixedRetPtrFree
+  | UnfixedRetPtrAddOffset
+  | UnfixedRetReadBadAlignment
+  | UnfixedRetWriteBadAlignment
+  | UnfixedRetDivRemByZero
```

Comment:
Would it be useful to capture the name of the function that did the returning for these?

Reply:
That's definitely reasonable, but I'm also planning on fixing all these right after #721 by using all the same heuristics for arguments, globals, and returned values of skipped functions (in fact, I have a WIP branch where that's done) so these will go away soon!